crates/rope/src/rope.rs

```diff
@@ -227,7 +227,7 @@ impl Rope {
         #[cfg(all(test, not(rust_analyzer)))]
         const PARALLEL_THRESHOLD: usize = 4;
         #[cfg(not(all(test, not(rust_analyzer))))]
-        const PARALLEL_THRESHOLD: usize = 4 * (2 * sum_tree::TREE_BASE);
+        const PARALLEL_THRESHOLD: usize = 1000;
```

Comment:
Let's keep this a multiple of tree base for better divisability 
```suggestion
        const PARALLEL_THRESHOLD: usize = 84 * (2 * sum_tree::TREE_BASE);
```
that comes out to = 1008